src/modules/baseCompare/baseCompare.js

```diff
@@ -0,0 +1,110 @@
+import GitHubAPI from 'github'
+import config from 'config'
+import ScreenshotBucket from 'server/models/ScreenshotBucket'
+
+async function fallbackToMaster(repository) {
+  const buckets = await ScreenshotBucket.query()
+    .where({
+      branch: 'master',
+      repositoryId: repository.id,
+    })
+    .orderBy('id', 'desc')
+  return buckets[0] || null
+}
+
+export default async function baseCompare(options) {
```

Comment:
We don't care that it's `options` or `foo` we just care about passing an object containing `baseCommit`, `compareCommit`, etc...